### PR TITLE
Release 0.2.3

### PR DESCRIPTION
**Pull request by Mr3zee**
Release of the 0.2.3 version:
- Update to K2 (2.0.0, 2.0.10)
- Support of 1.9.25
- APIs movements

This PR:
- Migration guide
- bump version
- update CHANGELOG.md

### CODE REVIEW

**Comment by Mr3zee** [comment]
@vnikolova can I please ask you to review `docs/pages/kotlinx-rpc/topics/0-2-3.topic`?

**Review by e5l — APPROVED** [review]

File: docs/pages/kotlinx-rpc/topics/0-2-3.topic [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+<topic xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="https://resources.jetbrains.com/writerside/1.0/topic.v2.xsd"
+       title="Migration to 0.2.3" id="0-2-3">
+
+    <p>
+        Version <code>0.2.3</code> does not bring breaking changes.
+        It introduces some, that may require additional changes in user projects.
+    </p>
+    <chapter title="Removal of KSP plugin for Kotlin 2.0" id="removal-of-ksp-plugin-for-kotlin-2-0">
+        If your project uses Kotlin 2.0 (stable versions of 2.0.0, 2.0.10 as of 0.2.3 release),
```
Comment (vnikolova) [inline docs/pages/kotlinx-rpc/topics/0-2-3.topic:13]: The text in parentheses can be removed.

File: docs/pages/kotlinx-rpc/topics/0-2-3.topic [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                id("org.jetbrains.kotlinx.rpc.plugin") version "0.2.3"
+
+                // KSP can be removed
+                // id("com.google.devtools.ksp") version "2.0.10-1.0.24"
+            }
+        </code-block>
+        This change brings one more benefit:
+        without KSP, projects that use <code>kotlinx.rpc</code>
+        can now use <a href="https://docs.gradle.org/current/userguide/configuration_cache.html">Gradle configuration
+        caches</a>
```
Comment (vnikolova) [inline docs/pages/kotlinx-rpc/topics/0-2-3.topic:28]:
```suggestion
        caches</a>.
```

File: docs/pages/kotlinx-rpc/topics/0-2-3.topic [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    </chapter>
+    <chapter title="Moving of client APIs" id="moving-of-client-apis">
+        Some client APIs were moved from <code>kotlinx-rpc-krpc-client</code> artifact (and package)
+        into <code>kotlinx-rpc-core</code> module (and package). These APIs are not kRPC specific hence the move.
+        List of APIs:
+        <list>
```
Comment (vnikolova) [inline docs/pages/kotlinx-rpc/topics/0-2-3.topic:34]: This list will look nicer in a [table](https://www.jetbrains.com/help/writerside/tables.html). You can have one table with two columns: `0.2.2` and `0.2.3` and the names under each.

**Review by vnikolova — CHANGES_REQUESTED** [review]
Looks good, just some minor tweaks in my comments.

**Review by vnikolova — APPROVED** [review]

**Comment by Mr3zee** [comment]
Closed in favor of #171

**Pull request closed without merging**